builtin/providers/openstack/resource_openstack_lbaas_listener_v2.go

```diff
@@ -0,0 +1,317 @@
+package openstack
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+
+	"github.com/rackspace/gophercloud"
+	"github.com/rackspace/gophercloud/openstack/networking/v2/extensions/lbaas_v2/listeners"
+)
+
+func resourceListenerV2() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceListenerV2Create,
+		Read:   resourceListenerV2Read,
+		Delete: resourceListenerV2Delete,
```

Comment:
@jtopjian I just realized, I missed adding the `Update: ` entry here!  But.. how could the test cases have passed if I missed this?  Fixing this in my next patch.